src/instructlab/model/serve.py

```diff
@@ -94,6 +94,13 @@ def signal_handler(
         "Automatically detected based on the model file properties.\n"
     ),
 )
+@click.option(
+    "--gpus",
+    type=click.IntRange(min=0),
```

Comment:
what's the meaning of `0` in this case? should it start from `1`? A brief look at `vllm` code suggests it won't work. (The "world_size" will end up as 0?)

Reply:
That's a good question I don't think has been asked yet.  This was recently changed from INT to IntRange(min=0) in multiple places.  You could argue 0 should imply CPU only.  But not saying it will currently work.  Probably should handle as a separate issue.

Reply:
The current implementation will do nothing, I think (`if args` will result in `False`.) But this may change. Wonder if we need to allow it in the first place.